src/main/java/SingleDriver.java

```diff
@@ -0,0 +1,38 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class SingleDriver {
+    private static WebDriver driver;
+
+    public static WebDriver getInstance() {
+        if (driver == null) {
+            synchronized (SingleDriver.class) {
+                if (driver == null) {
+                    WebDriverManager.chromedriver().setup();
+                    driver = new ChromeDriver();
+                    driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+                }
+            }
+        }
+        return driver;
+    }
+
+    public static void close() {
+        if (driver != null) {
+            driver.close();
```

Comment:
driver.close() just closes window, but driver != null after that. So you need to close and then assign null to driver

Reply:
This was changed, thank you!